### PR TITLE
JSCPD: Make it configurable via workflow input

### PR DESCRIPTION
**Pull request by jordicea**

### CODE REVIEW

**Comment by onurguzel** [comment]
What if someone else wants to use jscpd validation? Can you create an input for this variable, and pass it to the super-linter action?

**Comment by jordicea** [comment]
> What if someone else wants to use jscpd validation? Can you create an input for this variable, and pass it to the super-linter action?

Good point @onurguzel . Updated.

**Review by FelipeUmpierre — APPROVED** [review]

File: .github/workflows/linter.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ on:
         required: false
         type: string
         default: "main"
+      enable_jscpd:
```
Comment (onurguzel) [inline .github/workflows/linter.yaml:14]: It would be better if we preserve the wording, `validate_jscpd`. But not crucial.

**Review by onurguzel — APPROVED** [review]

File: .github/workflows/linter.yaml (hunk 1 above)
Comment (jordicea) [reply .github/workflows/linter.yaml:14]: I see more as on-off switch: validate_jscpd: enable(true-false|on-off) that's why I decided to put as `enable_jscpd`

**Pull request merged**